content/docs/start/experiments/experiment-tracking.md

```diff
@@ -38,9 +38,13 @@ There are some examples below
 from dvclive.lightning import DVCLiveLogger
 
 ...
-
-trainer = Trainer(logger=DVCLiveLogger(save_dvc_exp=True))
-trainer.fit(model)
+with Live(save_dvc_exp=True) as live:
```

Comment:
Let's also add `log_artifact` to the examples in https://dvc.org/doc/dvclive/ml-frameworks.

Reply:
This will take a while since I'm not that familiar with each of them. Adding this to your comment, and let's do this in a follow-up PR.